services/galley/test/integration/API/Federation.hs

```diff
@@ -73,45 +74,55 @@ tests s =
 
 getConversationsAllFound :: TestM ()
 getConversationsAllFound = do
-  -- FUTUREWORK: make alice / bob remote users
   [alice, bob] <- randomUsers 2
-  connectUsers alice (singleton bob)
-  -- create & get one2one conv
-  cnv1 <- responseJsonUnsafeWithMsg "conversation" <$> postO2OConv alice bob (Just "gossip1")
-  getConvs alice (Just $ Left [qUnqualified . cnvQualifiedId $ cnv1]) Nothing !!! do
-    const 200 === statusCode
-    const (Just [cnvQualifiedId cnv1]) === fmap (map cnvQualifiedId . convList) . responseJsonUnsafe
+  let aliceQ = Qualified alice (Domain "far-away.example.com")
+
   -- create & get group conv
-  carl <- randomUser
-  connectUsers alice (singleton carl)
-  cnv2 <- responseJsonUnsafeWithMsg "conversation" <$> postConv alice [bob, carl] (Just "gossip2") [] Nothing Nothing
-  getConvs alice (Just $ Left [qUnqualified . cnvQualifiedId $ cnv2]) Nothing !!! do
+  localDomain <- viewFederationDomain
+  carlQ <- Qualified <$> randomUser <*> pure localDomain
+  connectUsers bob (singleton (qUnqualified carlQ))
+
+  putStrLn $ "alice: " <> show (qUnqualified aliceQ)
+  putStrLn $ "bob: " <> show bob
+  putStrLn $ "carl: " <> show (qUnqualified carlQ)
+
+  cnv2 <-
+    responseJsonError
+      =<< postConvWithRemoteUser (qDomain aliceQ) (mkProfile aliceQ (Name "alice")) bob [aliceQ, carlQ]
```

Comment:
So, `alice` is no the local domain, but `aliceQ` is on `Domain "far-away.example.com"`, which might be different. Which domain of these two should go as the first argument to `postConvWithRemoteUser`?

Reply:
The remote one.